ui/src/components/command/CommandDialog.tsx

```diff
@@ -0,0 +1,160 @@
+import { Dialog } from '@headlessui/react';
+import { Command } from 'cmdk';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { selectCurrentNamespace } from '~/app/namespaces/namespacesSlice';
+
+interface Item {
+  name: string;
+  description: string;
+  onSelected: () => void;
+  keywords: string[];
+}
+
+interface CommandItemProps {
+  item: Item;
+}
+
+function CommandItem(props: CommandItemProps) {
+  const { item } = props;
+
+  return (
+    <Command.Item
+      value={item.name + ' ' + item.keywords?.join(' ')}
+      key={item.name}
+      className="flex cursor-pointer place-items-center px-4 py-2 data-[selected]:bg-violet-200 data-[selected]:dark:bg-gray-100"
+      onSelect={() => {
+        item.onSelected();
+      }}
+    >
+      <div className="flex flex-grow flex-col">
+        <span className="font-semibold">{item.name}</span>
+        <span className="text-gray-500 truncate text-xs">
+          {item.description}
+        </span>
+      </div>
+    </Command.Item>
+  );
+}
+
+export default function CommandMenu() {
+  const [open, setOpen] = useState(false);
+  const navigate = useNavigate();
+  const currentNamespace = useSelector(selectCurrentNamespace);
+
+  const namespacedItems = [
+    {
+      name: 'Flags',
+      description: 'Manage feature flags',
+      route: '/flags',
+      keywords: ['flags', 'flags', 'feature']
```

Comment:
```suggestion
      keywords: ['flags', 'flag', 'feature']
```

Reply:
Probably `flag` could be deleted as it will be covered with `flags`